Adapt provisioner for running local (#2918)
kyma-project/control-plane: components/provisioner/README.md

$ docker run --rm -p 5432:5432 -e POSTGRES_PASSWORD=password postgres
```

> Afterwards the database has to be created and migrations have to be run.

Reviewer: ```suggestion
Afterwards, the database must be created, and migrations must be run.
```

> Runtime Provisioner also needs a kubeconfig for a garden and auth data for Director.

Author: Both should be understandable. 
Gardener garden, where seeds are planted and shoots grow ;)

| APP_DIRECTOR_OAUTH_PATH                                       |                                                                                                           | `./dev/director.yaml`                                                   |
| APP_DIRECTOR_URL                                              | Director URL                                                                                              | `http://compass-director.compass-system.svc.cluster.local:3000/graphql` |
| APP_DOWNLOAD_PRE_RELEASES                                     |                                                                                                           | `true`                                                                  |
| APP_ENQUEUE_IN_PROGRESS_OPERATIONS                            | Specifies whether operations in the `InProgress` state should be enqueued on the application startup      | `true`                                                                  |

> | APP_GARDENER_AUDIT_LOGS_POLICY_CONFIG_MAP                     | Name of the Config Map containing the audit logs policy                                                   | optional                                                                |

Reviewer: ```suggestion
| APP_GARDENER_AUDIT_LOGS_POLICY_CONFIG_MAP                     | Name of the ConfigMap containing the audit logs policy                                                   | optional                                                                |
```
